Content.Server/Chemistry/EntitySystems/ChemistrySystem.cs

```diff
@@ -1,26 +0,0 @@
-using Content.Server.Administration.Logs;
-using Content.Server.Chemistry.Containers.EntitySystems;
-using Content.Server.Interaction;
-using Content.Server.Popups;
-using Content.Shared.Chemistry;
-using Robust.Shared.Audio.Systems;
-
-namespace Content.Server.Chemistry.EntitySystems;
-
-public sealed partial class ChemistrySystem : EntitySystem
-{
-    [Dependency] private readonly IAdminLogManager _adminLogger = default!;
-    [Dependency] private readonly IEntityManager _entMan = default!;
-    [Dependency] private readonly InteractionSystem _interaction = default!;
-    [Dependency] private readonly PopupSystem _popup = default!;
-    [Dependency] private readonly ReactiveSystem _reactiveSystem = default!;
-    [Dependency] private readonly SharedAudioSystem _audio = default!;
-    [Dependency] private readonly SolutionContainerSystem _solutionContainers = default!;
-
-    public override void Initialize()
-    {
-        // Why ChemMaster duplicates reagentdispenser nobody knows.
-        InitializeHypospray();
-        InitializeMixing();
```

Comment:
This needs to stay here (for now)

Reply:
Re-added ChemistrySystem as ReactionMixerSystem since the only interaction it ever had was the OnAfterInteract for... the bible, and isn't actually an all-chemistry encompassing system.